ios/AirshipFrameworkProxy/LiveActivity/LiveActivityManager.swift

```diff
@@ -96,8 +97,20 @@ public actor LiveActivityManager: Sendable {
         var update = false
         let all = self.entries.values.compactMap { try? $0.list() }.joined()
         for info in all {
+            do {
+                try track(
```

Comment:
This will result in multiple calls to `track` for the LA, but we handle spamming it on the SDK. We basically look for id/name combo and no-op if we are already watching for it